src/ui/components/layouts/base_layout/mobile_nav.rs

```diff
@@ -23,9 +23,9 @@ pub fn MobileNav() -> impl IntoView {
 fn NavLink(href: &'static str, icon: IconType, text: Signal<String>) -> impl IntoView {
   // TODO: Apply active to aria-current=page once the relevant Daisy UI issue is fixed: https://github.com/saadeghi/daisyui/issues/3170
   view! {
-    <A href=href>
+    <A href=href attr:class="aria-current-page:dock-active">
```

Comment:
While combining a variant with `dock-active` (the DaisyUI v5 equivalent to DaisyUI v4's `active`) no longer generates broken CSS, it still doesn't behave as expected. See the issue linked in the comment on line 24 for more info.